setup.py

```diff
@@ -24,8 +24,8 @@
         #   50.0 is broken: https://github.com/pypa/setupatools/issues/2353
         "setuptools>=41.0.0,!=50.0",
         "cached-property>=1.5.2",
-        "eclipse-sumo>=1.10.0",
         "click>=8.0.3",  # used in scl
+        "eclipse-sumo>=1.11.0",  # sumo

```

Comment:
Using `eclipse-sumo==1.11.0` previously threw an error message. See https://github.com/huawei-noah/SMARTS/issues/1155

Have we checked whether this issue is addressed? If not, I would suggest to use `eclipse-sumo==1.10.0` .

Reply:
actually the newest version of sumo is 1.12.0, given that issue 1155, maybe we shouldn't try 1.12.0 too?